### PR TITLE
Dok 143968

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
Update RL.

### Resolves:
* [OKTA-143968](https://oktainc.atlassian.net/browse/OKTA-143968)

### CODE REVIEW

File: _source/_docs/api/getting_started/design_principles.md [hunk 1]
```diff
@@ -306,56 +306,35 @@ endpoint.
...
+| `/api/v1/` (if no other limit specified in this table)                     |  1200 |
+| `/oauth2/v1/token`  (per second, not minute)                               |     4 |
+| `/oauth2/v1` (per second, not minute)                                      |    40 |
+
+For all API endpoints not listed in the table above, the API rate limit is a combined 10,000 requests per minute.
+
+
+| Okta End-User Endpoints                  | Limit |
+|:-----------------------------------------|------:|
+| `/app/{app}/{key}/sso/saml`              |   750 |
```
Comment (jmelberg-okta) [inline _source/_docs/api/getting_started/design_principles.md]: Is this an `appId`?

File: _source/_docs/api/getting_started/design_principles.md [hunk 2]
```diff
@@ -306,56 +306,35 @@ endpoint.
...
+| `/app/office365/{key}/sso/wsfed/active`  |  2000 |
+| `/app/office365/{key}/sso/wsfed/passive` |   250 |
+| `/app/template_saml_2_0/{key}/sso/saml`  |  2500 |
+| `/login/do-login`                        | 200   |
```
Comment (jmelberg-okta) [inline _source/_docs/api/getting_started/design_principles.md]: nit: These are left aligned vs all other limits being right aligned.

Comment (mystiberry-okta) [reply _source/_docs/api/getting_started/design_principles.md]: great catch, will fix before merge.

File: _source/_docs/api/getting_started/design_principles.md (hunk 1 above)
Comment (mystiberry-okta) [reply _source/_docs/api/getting_started/design_principles.md]: yikes, let me follow up with Evan. If it is, will fix.

**Pull request merged**